code/controllers/subsystem/shuttles/emergency.dm

```diff
@@ -97,7 +90,10 @@
 				send2irc("Server", "The Emergency Shuttle has docked with the station.")
 				priority_announce("The Emergency Shuttle has docked with the station. You have [timeLeft(600)] minutes to board the Emergency Shuttle.", null, 'sound/AI/shuttledock.ogg', "Priority")
 		if(SHUTTLE_DOCKED)
-			if(time_left <= 0)
+			if(time_left == 0 && SSshuttle.emergencyNoEscape)
```

Comment:
This ought to be `<= 0` as well, since time_left is not guaranteed to ever be 0.